src/markdown/mas/vergelijking_leefdaal.Rmd

```diff
@@ -836,20 +837,23 @@ opp_leefdaal_df <- leefdaal %>%
   mutate(oppervlakte = st_area(leefdaal)) %>%
   st_drop_geometry() %>%
   units::drop_units() %>%
-  arrange(naam)
+  arrange(naam) %>%
+  mutate(across(.cols = where(is.numeric), .fns = scale))
 
 # Landgebruiksamenstelling
 landgebruik_leefdaal_df <- landgebruik_leefdaal %>%
   pivot_wider(names_from = landgebruik_klasse, values_from = area_prop) %>%
   rename(Grasland_landgebruik = Grasland) %>%
-  arrange(naam)
+  arrange(naam) %>%
+  mutate(across(.cols = where(is.numeric), .fns = scales::rescale))
```

Comment:
This will rescale the areal proportions to a 0 - 1 scale by (x - min(x)) / (max(x) - min(x)). I doubt this is what you want for compositional data. I think what you will need is in `compositions::clr()`. In that case, you cannot use the `across()` construct because the transformation involves a standardization over columns (row sums become zero). After clr transformation, you can use `prcomp` and you will get the same results as `robCompositions::pcaCoDa(x, method = "classic")`


See https://cran.r-project.org/web/packages/compositions/vignettes/compositions_v2.html
for further info.